src/style.css

```
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  padding: 0;
+}
```

it's better not to add styles for all elements. leave `box-sizing` only
```suggestion
* {
  box-sizing: border-box;
}
```